prisma/migrations/20220708105026_submission_table_added_after_merge/migration.sql

```diff
@@ -0,0 +1,18 @@
+/*
+  Warnings:
+
+  - You are about to drop the `CohortExercise` table. If the table is not empty, all the data it contains will be lost.
+  - You are about to drop the `Submission` table. If the table is not empty, all the data it contains will be lost.
+
+*/
+-- DropForeignKey
+ALTER TABLE "Submission" DROP CONSTRAINT "Submission_cohortExerciseId_fkey";
+
+-- DropForeignKey
+ALTER TABLE "Submission" DROP CONSTRAINT "Submission_userId_fkey";
+
+-- DropTable
+DROP TABLE "CohortExercise";
```

Comment:
What is the reasoning for this?

Reply:
what should i do ?

Reply:
Sure. The point here is to double check schema changes, especially if they are lossy.

In this case, this change shouldn't be here - unless you have a reason to include it.